packages/rtmpdump.rb

```diff
@@ -20,20 +21,12 @@ class Rtmpdump < Package
         i686: '109fb6fa70409f8ba274fa452c2e04dc1ff3ba740f618525c188139c1ddce363',
       x86_64: '0212408c6faad92b5909d77188d8c27e68ce955f61dcb5597603303e31e601ce',
   })
-  
-  def self.patch
-    system "sed -i 's,prefix=/usr/local,prefix=#{CREW_PREFIX},' Makefile"
-    system "sed -i 's,prefix=/usr/local,prefix=#{CREW_PREFIX},' librtmp/Makefile"
-    system "sed -i 's,libdir=\$(prefix)/lib,libdir=#{CREW_LIB_PREFIX},' librtmp/Makefile"
-    # OpenSSL builds are broken.
-    system "sed -e 's/^CRYPTO=OPENSSL/#CRYPTO=OPENSSL/' -e 's/#CRYPTO=GNUTLS/CRYPTO=GNUTLS/' -i Makefile -i librtmp/Makefile"
-  end
- 
+
   def self.build
-    system 'make'
+    system 'make', *@make_common_opts
```

Comment:
We have a constant for this already.  Try `system "make #{CREW_OPTIONS} CRYPTO=GNUTLS"`.

Reply:
This Makefile does not follow the normal conventions. prefix and libdir are lower-case, so CREW_OPTIONS is not appropriate here.